src/webhooks/registry.ts

```diff
@@ -384,6 +388,9 @@ const WebhooksRegistry: RegistryInterface = {
         if (!domain) {
           missingHeaders.push(ShopifyHeader.Domain);
         }
+        if (!webhookId) {
+          missingHeaders.push(ShopifyHeader.WebhookId);
+        }
```

Comment:
Since this header isn't strictly necessary for validation, I wonder if we want to fail on it missing. That being said, we should always have the header so I suppose there's no harm in checking 🙂.

Reply:
It's optional in the `WebhookHandlerFunction` type, so it probably shouldn't be mandatory here.  The existing tests aren't passing because of this check

Reply:
My 2c: It's optional in the type for backwards compatibility but the expected behavior is the header always exists. Therefore, I'd argue this is _more_ correct. However, it's a minor nit and it probably doesn't matter much either way. Happy to go either direction!